### PR TITLE
Feature #179 data store 

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: DataStore/interfaceview.xml [hunk 1]
```diff
@@ -0,0 +1,283 @@
+<InterfaceView version="1.1" asn1file="datastore.asn">
```
Comment (Lurkerpas) [inline DataStore/interfaceview.xml]: I think I am missing publish/subscribe interfaces. We can discuss this if needed.

File: DataStore/datastore.asn [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+   DataStoreInternalLogStorage ::= SEQUENCE (SIZE(data-store-log-size)) OF DataStoreInternalLogItem
+
+   T-EventMessage ::= CHOICE {
+      item-created SEQUENCE {
+         item-key DataStoreKeyType,
+         item-timestamp ULongInteger
+      },
+      item-retrieved SEQUENCE {
+         item-key DataStoreKeyType,
+         item-value DataStoreValueType,
```
Comment (Lurkerpas) [inline DataStore/datastore.asn]: Extract retrieved events into a separate interface

Comment (rphh) [reply DataStore/datastore.asn]: done

File: DataStore/interfaceview.xml (hunk 1 above)
Comment (rphh) [reply DataStore/interfaceview.xml]: subscribe_to_event was implemented, but I have issues with running the project

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**